packages/lit-html/src/lit-html.ts

```diff
@@ -233,9 +237,10 @@ export const render = (
 ) => {
   let part: NodePart = (container as any).$lit$;
   if (part === undefined) {
-    const marker = createMarker();
-    container.append(marker);
-    (container as any).$lit$ = part = new NodePart(marker, null, options);
+    const start = createMarker();
+    const end = options?.refNode || null;
+    container.insertBefore(start, end);
+    (container as any).$lit$ = part = new NodePart(start, end, options);
```

Comment:
Since Terser won't know that insertBefore returns the new node, we can manually inline it and save a couple of bytes when it (hopefully) inlines the `start` variable because it's only used once:

```ts
(container as any).$lit$ = part = new NodePart(container.insertBefore(start, end), end, options);
```

Reply:
Done.